### PR TITLE
Fixing a content issue in hello-world.yml

### PR DESCRIPTION
**Pull request by shahriaarrr**
In the explanation of this document about assigning values, variables were given to that value without defining a variable.

Wrong ❌
```
aFriend = "Maira";
Console.WriteLine(aFriend);
```

correct ✔
```
string aFriend;
aFriend = "Maira";
Console.WriteLine(aFriend);
```

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
Thanks for opening the PR @shahriaarrr. However, I think the original content is correct.

Previously in the article, the variable `aFriend` was declared and printed like the following:

```csharp
string aFriend = "Bill";
Console.WriteLine(aFriend);
```

Then, the article shows that you can *modify* a variable that's already declared before.

Note the statement before this code snippet saying:

> Add these two lines in the interactive window **following the code you've already added** (emphasis mine).

You can wait for an "official" review from the .NET docs team in case someone can suggest any possible improvements.

**Comment by shahriaarrr** [comment]
Thanks @Youssef1313  for the message you gave and for pointing me to this point. But I think this modification will somehow make this document more accurate and understandable, especially for those who are just starting out with C # and the .NET Framework.

**Comment by Youssef1313** [comment]
The snippet is intended to show that you can modify the previously declared variable. So I think making it more clear that this snippet is intended to be written below the previous declaration and usage is a better option than changing the snippet (which is actually more confusing with the existing text IMO).

@BillWagner What do you think?

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml
Comment (BillWagner) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:64]: Let's remove this, based on our discussion and the suggestion above.

```suggestion
```

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml
Comment (BillWagner) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:60]: Let's make this more clear.  (GitHub won't let me apply a suggestion to all three lines, so we may need to edit by hand in addition to applying this suggestion):

```suggestion
    the name to one of your friends. Add these two lines in the interactive window
    following the code you've already added.  Make sure you keep the declaration
    of the `aFriend` variable and its initial assignment.
```

File: docs/csharp/tour-of-csharp/tutorials/hello-world.yml
Comment (BillWagner) [inline docs/csharp/tour-of-csharp/tutorials/hello-world.yml:61]: Remove this line that was added and modified in my suggestion above:

```suggestion
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for making this update @shahriaarrr and for the discussion @Youssef1313 

I've made some suggestions that should keep the initial intent, and clarify the edits at this point in the tutorial.

Let me know what you think, and once we resolve this, it will be ready to ship.

**Review by Youssef1313 — APPROVED** [review]
LGTM with @BillWagner suggestions.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks again @shahriaarrr and @Youssef1313 

I'll :shipit: as soon as the CI build finishes.

You should see the changes on our live site in the next day or two, on our regular publishing cycle.

**Comment by Youssef1313** [comment]
@gewarren FYI the status checker shows `Error: Did not find OPS status check. Please close/reopen the pull request.` when the OPB hasn't yet finished. I thought you fixed that few days ago, not sure what's going on.

**Comment by shahriaarrr** [comment]
Oh ni. @BillWagner  we have some problem here

**Comment by BillWagner** [comment]
closing and reopening for a fresh build.

**Comment by gewarren** [comment]
> @gewarren FYI the status checker shows `Error: Did not find OPS status check. Please close/reopen the pull request.` when the OPB hasn't yet finished. I thought you fixed that few days ago, not sure what's going on.

Thanks for letting me know. However, this is actually the desired behavior when OpenPublishing.Build is still _Expected_ after a few minutes. The recommended action is to close/reopen the PR to retrigger the build. I've updated the messaging in status-checker to hopefully make that clearer in https://github.com/dotnet/samples/pull/4563.

**Pull request merged**
